utils/Bootstrap.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jest/no-conditional-expect */
+/*
+ *
+ * @group bootstrap
+ * @group sequential
+ */
+import { getApi } from "./api";
+import { finalizeBootstrap } from "./tx";
+import { EventResult, ExtrinsicResult } from "./eventListeners";
+import { User } from "./User";
+import { getEventResultFromMangataTx, sudoIssueAsset } from "./txHandler";
+import { BN, toBN } from "@mangata-finance/sdk";
+import { Assets } from "./Assets";
+import { setupApi } from "./setup";
+import { Sudo } from "./sudo";
+
+export async function createNewBootstrapToken(sudoUser: User) {
+  const api = getApi();
+  await setupApi();
+
+  let bootstrapPhase: any;
+  let eventResponse: EventResult;
+
+  // check that system is ready to bootstrap
+  bootstrapPhase = await api.query.bootstrap.phase();
+  if (bootstrapPhase.toString() === "Finished") {
+    const bootstrapFinalize = await finalizeBootstrap(sudoUser);
+    eventResponse = getEventResultFromMangataTx(bootstrapFinalize);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  }
+
+  bootstrapPhase = await api.query.bootstrap.phase();
+  expect(bootstrapPhase.toString()).toEqual("BeforeStart");
```

Comment:
Is this required to createABootstrapToken?

Reply:
I decided to keep this checking because it guarantees that we don't try to prepare new bootstrap before we finish the last one.  I'm not sure about that decision, I can delete this or rename the function to prepareBootstrap or prepareBootstrapCurrency